utils/syntax_highlighting.tsx

```diff
@@ -2,133 +2,10 @@
 // See LICENSE.txt for license information.
 
 import hlJS from 'highlight.js';
-import hljsActionscript from 'highlight.js/lib/languages/actionscript.js';
```

Comment:
Are languages automatically enabled for highlighting now? If so, we might be able to close a help wanted ticket where we wanted to lazy load individual languages.

Reply:
Yes they are automatically enabled now. You can still change it to a mode where the languages can be imported individually, so that ticket is probably still possible.

Reply:
I wonder what the impact of that on the bundle size will be. I can't imagine it's actually too much to include some new languages even compared to entirely new libraries like we add frequently.

As for the lazy loading ticket, I think I wrote that in a way that it won't need to be updated for this, but I did link to this PR there in case it changes things.